examples/Processor.hs

```diff
@@ -120,7 +148,7 @@ read k = liftSelect (R k id)
 
 -- | A convenient alias for writing into an element of the processor state.
 write :: Key -> Program Value -> Program Value
-write k fv = fv *> liftSelect (W k fv id)
+write k fv = liftSelect (W k fv id) -- fv *> liftSelect (W k fv id)
```

Comment:
Let's drop the comment, since it looks like our previous approach was just wrong.

Reply:
Agreed.